### PR TITLE
Adding recharts initial commit

### PR DESCRIPTION
**Pull request by jhalaa**

### CODE REVIEW

File: src/components/DisplayComponent.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend
+} from 'recharts';
 import '../styles/DisplayComponent.css';
 import Filters from './Filters';
+import testTable from '../../src/testStuff/cities.json';
```
Comment (steventhan) [inline src/components/DisplayComponent.js]: This json data is not used anywhere, yet

File: src/components/DisplayComponent.js [hunk 2]
```diff
@@ -14,6 +24,33 @@ class DisplayComponent extends Component {
...
 
   render() {
+    var display;
+    if (this.state.displayComponent === 'Chart') {
+      display = (
+        <BarChart
+          width={600}
+          height={300}
+          data={[
+            { name: 'Page A', uv: 4000, pv: 2400, amt: 2400 },
```
Comment (steventhan) [inline src/components/DisplayComponent.js]: I'm assuming these are just placeholder data, and will be removed as we move forward.

File: src/components/DisplayComponent.js [hunk 3]
```diff
@@ -14,6 +24,33 @@ class DisplayComponent extends Component {
...
+            { name: 'Page B', uv: 3000, pv: 1398, amt: 2210 },
+            { name: 'Page C', uv: 2000, pv: 9800, amt: 2290 },
+            { name: 'Page D', uv: 2780, pv: 3908, amt: 2000 },
+            { name: 'Page E', uv: 1890, pv: 4800, amt: 2181 },
+            { name: 'Page F', uv: 2390, pv: 3800, amt: 2500 },
+            { name: 'Page G', uv: 3490, pv: 4300, amt: 2100 }
+          ]}
+          margin={{ top: 5, right: 30, left: 20, bottom: 5 }}
```
Comment (steventhan) [inline src/components/DisplayComponent.js]: I see you do use inline css. That case, we can use both `scss` and `jss` lib in parallel

File: src/components/DisplayComponent.js [hunk 4]
```diff
@@ -14,6 +24,33 @@ class DisplayComponent extends Component {
   }
 
   render() {
+    var display;
+    if (this.state.displayComponent === 'Chart') {
+      display = (
```
Comment (steventhan) [inline src/components/DisplayComponent.js]: This if/else stmt is getting large, can you break it down into smaller helper components say `<Chart />` and `<Map />` components?

File: src/components/DisplayComponent.js (hunk 1 above)
Comment (jhalaa) [reply src/components/DisplayComponent.js]: Ok! Will remove!

File: src/components/DisplayComponent.js (hunk 4 above)
Comment (jhalaa) [reply src/components/DisplayComponent.js]: Good point! Will do that. Thank you

File: src/components/DisplayComponent.js (hunk 2 above)
Comment (jhalaa) [reply src/components/DisplayComponent.js]: yes! this is just placeholder data. I will remove it once I tie it up with the api results

File: src/components/DisplayComponent.js (hunk 3 above)
Comment (jhalaa) [reply src/components/DisplayComponent.js]: This is not inline css. This is a prop to the Barchart component in recharts. If the props margin is set then the component is setting the margin css. I think this is a cleaner way to set css. I you feel we should move it elsewhere do let me know! I'd be happy to do that.

**Review by steventhan — APPROVED** [review]
Great!

**Pull request merged**